test_util/src/lib.rs

```diff
@@ -288,3 +290,25 @@ where
         }
     }
 }
+
+/// Install a tracing subscriber for use in unit tests. This should be called at the beginning
```

Comment:
super-nit: this is not exclusively for _unit_ tests
```suggestion
/// Install a tracing subscriber for use in tests. This should be called at the beginning
```